### DISCUSSION TITLE
User timeout on multiple sessions

### DISCUSSION THREAD

**Discussion by RedBlob** [Budibase/budibase#12173] [Help] [answered]
Hello,
Where is possible to find documentation about : 
- Maximum number of simultaneous users with the same account ( same e-mail ) on the same app but from different devices
- Disconnetiction timeout : after how many minutes a user who is not active on the web page is disconnected or deactivated from the maximum number of active connections of the same user 
Thank You
Best Regards

**Comment by RedBlob** [comment]
Hello,
Any suggestions?

**Reply by ConorWebb96** [reply, marked as answer]
Hey @RedBlob,

- You can only be logged in with a single user at a time as well as only be logged in on one device with that single user. If you try to login again elsewhere the previous instance will be logged out.
- The logout timer is a week.

I hope this helps.
